Linux_android spell_check_test is consistently failing, "The processing instruction target matching "[xX][mM][lL]" is not allowed."

```
[spell_check_test] [STDOUT] stderr: [        ] FAILURE: Build failed with an exception.
[spell_check_test] [STDOUT] stderr: [        ] * What went wrong:
[spell_check_test] [STDOUT] stderr: [        ] Execution failed for task ':app:mergeDebugResources'.
[spell_check_test] [STDOUT] stderr: [        ] > /opt/s/w/ir/x/w/recipe_cleanup/tmpg4n5wqkj/flutter sdk/dev/integration_tests/spell_check/android/app/src/main/res/values/styles.xml:5:6: Error: The processing instruction target matching "[xX][mM][lL]" is not allowed.
[spell_check_test] [STDOUT] stderr: [        ] * Try:
```
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8804378514533463825/+/u/run_spell_check_test/test_stdout

@camsim99 testowner

--- a/dev/integration_tests/spell_check/android/app/src/main/res/drawable/launch_background.xml
+++ b/dev/integration_tests/spell_check/android/app/src/main/res/drawable/launch_background.xml
@@ -1,8 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright 2014 The Flutter Authors. All rights reserved.
 Use of this source code is governed by a BSD-style license that can be
 found in the LICENSE file. -->
 
-<?xml version="1.0" encoding="utf-8"?>
 <!-- Modify this file to customize your launch splash screen -->
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:drawable="@android:color/white" />

